operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakDeploymentTest.java

```diff
@@ -435,9 +471,9 @@ public void testHttpRelativePathWithSecretValue() {
 
             kc.getSpec().getServerConfiguration().add(new ValueOrSecret(Constants.KEYCLOAK_HTTP_RELATIVE_PATH_KEY,
                     new SecretKeySelectorBuilder()
-                        .withName(secretName)
-                        .withKey(keyName)
-                        .build()));
+                            .withName(secretName)
+                            .withKey(keyName)
+                            .build()));
```

Comment:
I believe this change is out-of-scope of this PR. Please revert if possible.

Reply:
Done! Thanks for your review and comment.